integration_tests/cairozero_test.go

```diff
@@ -67,6 +125,9 @@ func TestCairoZeroFiles(t *testing.T) {
 
 	benchmarkMap := make(map[string][2]int)
 
+	sem := make(chan struct{}, 5) // semaphore to limit concurrency
```

Comment:
So right now only 5 goroutines can be run simultaneously? Is that number arbitrary? Did you test bigger / smaller options and how they affect performance?

Reply:
i think Hari said he wanted to start with only a few, see how CI behaves, and then increment?

Reply:
It's kinda arbitrary. I didn't see a huge improvement after 5 goroutines and wanted to be conservative for our CI.